### PR TITLE
CI-129 - generalised the manage apprenticeship funds link into a part…

### PR DESCRIPTION
**Pull request by davehowlett**
…ial view.

### CODE REVIEW

File: src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Views/Provider/StandardResultsPage.cs [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-            {
-                IsLevyPayingEmployer = false,
-                Hits = new List<StandardProviderResultItemViewModel>()
-            };
-            var html = detail.RenderAsHtml(model).ToAngleSharp();
-
-            this.GetPartial(html, "#register_to_manage").Should().BeEmpty();
-        }
-    }
-}
```
Comment (MarkFCain) [inline src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Views/Provider/StandardResultsPage.cs]: Why did you remove these tests?

Comment (davehowlett) [reply src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Views/Provider/StandardResultsPage.cs]: Originally, I had repeated the html in the standardresults, Frameworkresults and Deatilas views. As they were in actual views, I could use AngleSharp to parse the whole document and write tests accordingly.

When I generalised them all into a partial view, I found that there wasn't a way for the HttpAgilityPack to render just that bit of HTML with the VM, and so I couldn't write tests for it.

Comment (MarkFCain) [reply src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Views/Provider/StandardResultsPage.cs]: Fair enough!
